### ISSUE TITLE
Device is busy or does not respond.

### ISSUE THREAD

**Issue by tbellp** [thonny/thonny#2019]
Device is busy or does not respond. Your options:

  - wait until it completes current work;
  - use Ctrl+C to interrupt current work;
  - use Stop/Restart to interrupt more and enter REPL.

No prompt appears
Linux 18.04
Micropython on ESP32 flashed
thonny version 3.3.14

**Comment by aivarannamaa** [comment]
Please reopen if you have more details to add!

**Comment by tbellp** [comment]
> Can you post your main.py and boot.py?
Flashed with esp32-20210902-v1.17.bin option selected: erase flash before installing 
Just flash the .bin from micropython.org with esptool. I tried other flashing with thonny, but console did not show '>>>' insted device is busy or does not respond. Your options:
 wait until it completes current work;
  - use Ctrl+C to interrupt current work;
  - use Stop/Restart to interrupt more and enter REPL.
 I tried those choices but nothing happen.
I tried with other ESP - F1, old flashed with micropython and it happened the same problem.
ESP has communication, but console can't interprete ESP data.
